From 56aa426f529693038ef6dc0a050191d1f035b0dd Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 24 Oct 2016 18:08:39 +0800
Subject: [PATCH 1233/5242] MLK-13384 ARM: imx: remove ldo bypass check on
 imx6sll

commit  bc364c71312378d4d70e49d078e74c4d1be6eb93 from
https://source.codeaurora.org/external/imx/linux-imx.git

As on i.MX6SLL, there is no ARM LDO, the code for ARM LDO
bypass check is unnecessary, remove these piece of code in
i.MX6SLL low power idle.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/imx6sll_low_power_idle.S |   12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm/mach-imx/imx6sll_low_power_idle.S b/arch/arm/mach-imx/imx6sll_low_power_idle.S
index e18cc39..a7e206e 100644
--- a/arch/arm/mach-imx/imx6sll_low_power_idle.S
+++ b/arch/arm/mach-imx/imx6sll_low_power_idle.S
@@ -210,12 +210,6 @@
 	bic	r7, r7, #0x1
 	str	r7, [r10, #0x110]
 
-	/* check whether ARM LDO is bypassed */
-	ldr	r7, [r10, #0x140]
-	and	r7, r7, #0x1f
-	cmp	r7, #0x1f
-	bne	10f
-
 	/* low power band gap enable */
 	ldr	r7, [r10, #0x270]
 	orr	r7, r7, #0x20
@@ -302,12 +296,6 @@
 	ands	r7, r7, #0x40000
 	beq	11f
 
-	/* check whether ARM LDO is bypassed */
-	ldr	r7, [r10, #0x140]
-	and	r7, r7, #0x1f
-	cmp	r7, #0x1f
-	bne	12f
-
 	/* turn on regular bandgap and wait for stable */
 	ldr	r7, [r10, #0x150]
 	bic	r7, r7, #0x1
-- 
1.7.9.5

